libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -807,6 +807,7 @@ class MapboxRouteLineApi(
         val trafficLineExpressionProducer = partitionedRoutes.first.firstOrNull()?.route?.run {
             MapboxRouteLineUtils.getTrafficLineExpressionProducer(
                 this,
+                routeLineOptions.resourceProvider.congestionRange,
```

Comment:
I think there's a missed opportunity here. Ultimately a range equals a color. Couldn't you combine `CongestionRange` and `RouteLineColorResources` in some way?  Seems more efficient if a developer configures these together for example the low traffic range is from a..b with color yellow, moderate traffic is range b..c with color orange etc.